libs/features/advert-page/src/lib/features-advert-page.tsx

```diff
@@ -23,13 +23,15 @@ export const FeaturesAdvertPage = () => {
           description={advert.description}
           creator={advert.creator.name}
           image={advert.imageUrl}
-          categories={[{ title: advert.category, value: advert.category }]}
+          categories={[]}
+          allCategoriesList={[]}
```

Comment:
Is categories still needed then?
Why is it empty?

Reply:
Returned it back